README.md

```diff
@@ -9,10 +9,12 @@ TelegramClient is a **Squeak-Client** for the widely used Telegram-Messenger. It
 
 # Installation
 Make sure you have the current version of Squeak installed.
-Step 1: Install the Squeak Git Client
-Step 2: Install the Squeak FFI Workload
-Step 3: Download the required native Binaries for your system. ![Windows](https://drive.google.com/open?id=14tOoIfxioIe8bn2kPxPKmCk_Eg0tSYiJ) ![Mac](https://drive.google.com/open?id=1H2l_V6zkIx4t_SKQKL3MmdtvOV2haFjV) ![Ubuntu 18 lts](https://drive.google.com/open?id=1CpSPcB-5MPA95fEWI-X5MxbUEZUcU3g2).
-Step 4: Place the unzipped binaries inside of your squeak image. On Windows, place them inside the Win32 folder, on Linux and Mac Os place them inside the resources folder.
+* Step 1: Install the Squeak Git Client
+* Step 2: Install the Squeak FFI Workload. To do so, open the Workspace in Squeak and execute the following command:   `(Installer repository: 'http://source.squeak.org/FFI')
+install: 'FFI-Pools';
+install: 'FFI-Kernel'.`
+* Step 3: Download the required native Binaries for your system. [Windows](https://drive.google.com/open?id=14tOoIfxioIe8bn2kPxPKmCk_Eg0tSYiJ), [Mac](https://drive.google.com/open?id=1H2l_V6zkIx4t_SKQKL3MmdtvOV2haFjV), [Ubuntu 18 lts](https://drive.google.com/open?id=1CpSPcB-5MPA95fEWI-X5MxbUEZUcU3g2).
+* Step 4: Place the unzipped binaries inside of your squeak image. On Windows, place them inside the Win32 folder, on Linux in "Contents/Linux-x86_64/lib/squeak/\<versionnumber>/" and on Mac place them inside the resources folder.
 # Running TelegramClient
 In a workspace window execute the command 
 # Building
```

Comment:
Building instructions still contain image link from previous edit

Reply:
+ Installation instructions are not complete/working for the Windows platform, after all steps have been performed the error "Module not found" occured
Perhaps some files or runtime modules missing

Reply:
As discussed in the Telegram group, this probably happens due to missign msvcrt14 installation. As nobody answered to my call to please investigate whether this is the root cause this was dropped - if this happens please refer to the posts I made there and check whether that solution works. If it indeed does, we can add that to our installation instructions.
Also, this does not mean that the installation instructions are wrong. If I were to follow them it would work on my machine, just as it does not work on yours.